README.md

```diff
@@ -77,6 +77,34 @@ var updateSourceData = function(source) {
 videojs.Html5DashJS.hook('updatesource', updateSourceData);
 ```
 
+## Captions
+
+As of `video.js@5.14`, native captions are no longer supported on any browser besides Safari. Dash can handle captions referenced embedded vtt files, embedded captions in the manifest, and with fragmented text streaming. It is impossible to use video.js captions when dash.js is using fragmented text captions, so the user must disable native captions when using `videojs-contrib-dash`.
+
+```javascript
+videojs('example-video', {
+  html5: {
+    nativeCaptions: false,
+  },
+});
+```
+
+A warning will be logged if this setting is not applied.
+
+### Automatic Track Selection
+
+`videojs-contrib-dash` will listen for `video.js`'s built-in text track change events and will update the dash.js text track automatically.
+
+If the user wants `videojs-contrib-dash` to load a list of the text tracks but then to allow the user to programatically respond to `video.js`'s built-in text track change events, then pass the `automaticTextTrackSelection` option:
+
+```javascript
+videojs('example-video', {
+  html5: {
+    automaticTextTrackSelection: false,
```

Comment:
That is very close to what I'm doing, my logic was to prevent both `videojs-contrib-dash` and my application from trying to set the active dash track. I don't know if that's necessary though.